tests/update/patch_before_migration.pm

```diff
@@ -73,13 +76,46 @@ sub patching_sle {
     set_var("SYSTEM_PATCHED", 1);
 }
 
+sub install_patterns {
+    select_console 'root-console';
+    my $pcm = 0;
+    my @pt_list;
+    # install all patterns from product.
+    if (check_var('PATTERNS', 'all')) {
+        @pt_list = split(/ /, script_output("zypper pt -u | grep '^ ' | tr -s ' ' | cut -d ' ' -f 3 | tr -s '\n' ' '"));
+    }
```

Comment:
Do we really need this? It looks waist time. From the ticket, I think we have done all patterns (base+extensions) installation during image creation, what we to do here is just intall petterns for modules.

Reply:
"zypper pt -u" will show only patterns which are not installed, so don't worry about it.
I will improve this block and reduce duplicate patterns.